DeepFried2/layers/Dimshuffle.py

```diff
@@ -0,0 +1,10 @@
+import DeepFried2 as df
+
+
+class Dimshuffle(df.Module):
+    def __init__(self, *new_shape):
+        df.Module.__init__(self)
+        self.new_shape = new_shape
+
+    def symb_forward(self, symb_input):
+        return symb_input.dimshuffle(self.new_shape)
```

Comment:
Aren't you missing a `*` here?

`self.new_shape` is a tuple, but according to documentation, `dimshuffle` takes "splatted" arguments, not a tuple.